cvat/apps/iam/templates/account/email/email_confirmation_message.html

```diff
@@ -107,12 +107,6 @@
 </head>
 <body style="background-color: #e9ecef;">
 
-  <!-- start preheader -->
-  <div class="preheader" style="display: none; max-width: 0; max-height: 0; overflow: hidden; font-size: 1px; line-height: 1px; color: #fff; opacity: 0;">
-    A preheader is the short summary text that follows the subject line when an email is viewed in the inbox.
```

Comment:
Please insert something like: "Get Started with CVAT: Watch Product Tour, Explore Documentation, and Enjoy Exclusive Offer - Your Smooth Start Awaits!"

Reply:
Yes, I agree probably we need to have a nice-looking preheader for the Welcome letter and I have added your suggestion in a separate PR.

But this PR removes the preheader for the email confirmation letter and users will see the following sentence "Confirm Your Email Address Thank you for signing up for CVAT!"